Handreiking API Design/Handreiking APi Design.md

```diff
@@ -71,19 +87,32 @@ naamgevingsconventie voor de relatieklasse in het recessieve domein:
 
 Voorbeelden hiervan zijn:
 
-* `ObjectContactmoment`
-* `ObjectInformatieobject`
+* `objectverzoek`
+* `objectcontactmoment`
+* `objectinformatieobject`
 
 In dit geval is er ook geen sprake van twee relatieklasses met dezelfde naam in
 het gegevensmodel. Een goed voorbeeld van 2 van deze _tightly coupled_ klasses
-met verschillende namen is `ZaakContactmoment` en `ObjectContactmoment`. In dit
+met verschillende namen is `zaakcontactmoment` en `objectcontactmoment`. In dit
 geval zal in zo'n klasse die koppelt aan meerdere domeinen ook een attribuut
 worden opgenomen waarmee het type van de te koppelen klassse aangegeven kan
 worden.
 
 **Kardinaliteit van deze classes**
 
-...
+Tussen een entiteit en de relatie-entiteit in een andere domein die de entiteit daar representeert is altijd een 1 op 1 koppeling van toepassing. De kardinaliteit aan beide zijde van deze relatie is dus maximaal 1. Aangezien een entiteit in een ander domein niet per definitie hoeft te leiden tot een relatie-entiteit moet de kardinaliteit van de relatie tussen de relatie-entiteit en de entiteit in het andere domein aan de relatie-entiteit zijde (C en G in de onderstaande illustratie) altijd 0..1 zijn. Aan de entiteit zijde (D en H  in de onderstaande illustratie) is deze niet optioneel en dus exact 1.
```

Comment:
Ik denk dat ik snap wat je bedoelt maar hebben we voorbeelden van relaties die maar vanuit één domein te benaderen zijn?

Reply:
Ik op mijn beurt begrijp niet wat je bedoelt...;-)
Maar waarschijnlijk is dat niet meer zo van belang. Ook in deze sectie ga ik waarschijnlijk wijzigingen aanbrengen n.a.v. voortschrijdend inzicht.